datumaro/components/dataset.py

```diff
@@ -834,6 +881,16 @@ def flush_changes(self):
     @scoped
     def export(self, save_dir: str, format: Union[str, Type[Extractor]],
             **kwargs) -> None:
+        """
+        Saves the input dataset in some format.
```

Comment:
```suggestion
        Saves the dataset in some format.
```

IMO, "input dataset" doesn't really make sense here, since you don't pass a dataset explicitly into this function.

Reply:
Fixed